# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../cdrdao/gcc43.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Naurp cdrdao-1.2.2.orig/trackdb/FormatConverter.cc cdrdao-1.2.2/trackdb/FormatConverter.cc
--- cdrdao-1.2.2.orig/trackdb/FormatConverter.cc	2005-05-10 00:55:28.000000000 +0000
+++ cdrdao-1.2.2/trackdb/FormatConverter.cc	2008-03-07 11:14:07.000000000 +0000
@@ -22,6 +22,7 @@
 #include <ao/ao.h>
 #endif
 #include <fstream>
+#include <cstring>
 
 #include "config.h"
 #include "util.h"
diff -Naurp cdrdao-1.2.2.orig/trackdb/FormatOgg.cc cdrdao-1.2.2/trackdb/FormatOgg.cc
--- cdrdao-1.2.2.orig/trackdb/FormatOgg.cc	2005-04-22 02:01:46.000000000 +0000
+++ cdrdao-1.2.2/trackdb/FormatOgg.cc	2008-03-07 11:14:07.000000000 +0000
@@ -18,6 +18,7 @@
  */
 
 #include <stdio.h>
+#include <cstring>
 
 #include "util.h"
 #include "FormatOgg.h"
diff -Naurp cdrdao-1.2.2.orig/trackdb/FormatMp3.cc cdrdao-1.2.2/trackdb/FormatMp3.cc
--- cdrdao-1.2.2.orig/trackdb/FormatMp3.cc	2005-09-24 20:28:43.000000000 +0000
+++ cdrdao-1.2.2/trackdb/FormatMp3.cc		2008-03-07 11.14.07.000000000 +0000
@@ -25,6 +25,7 @@
 #include <stdlib.h>
 #include <unistd.h>
 #include <fcntl.h>
+#include <cstring> 
 #include <sys/types.h>
 #include <sys/stat.h>
 #include <sys/mman.h>
diff -Naurp cdrdao-1.2.2.orig/trackdb/TempFileManager.cc cdrdao-1.2.2/trackdb/TempFileManager.cc
--- cdrdao-1.2.2.orig/trackdb/TempFileManager.cc	2005-04-22 02:01:46.000000000 +0000
+++ cdrdao-1.2.2/trackdb/TempFileManager.cc	2008-03-07 11:14:07.000000000 +0000
@@ -24,6 +24,7 @@
 #include <sys/stat.h>
 #include <unistd.h>
 #include <fcntl.h>
+#include <cstring>
 
 #define DEFAULT_TEMP_PATH "/tmp/"
 
diff -Naurp cdrdao-1.2.2.orig/xdao/AudioCDProject.cc cdrdao-1.2.2/xdao/AudioCDProject.cc
--- cdrdao-1.2.2.orig/xdao/AudioCDProject.cc	2006-09-18 10:42:26.000000000 +0000
+++ cdrdao-1.2.2/xdao/AudioCDProject.cc	2008-03-07 11:15:26.000000000 +0000
@@ -18,6 +18,7 @@
  */
 
 #include <assert.h>
+#include <cstring>
 #include <gtkmm.h>
 #include <libgnome/gnome-i18n.h>
 
diff -Naurp cdrdao-1.2.2.orig/xdao/TextEdit.cc cdrdao-1.2.2/xdao/TextEdit.cc
--- cdrdao-1.2.2.orig/xdao/TextEdit.cc	2004-02-12 01:13:32.000000000 +0000
+++ cdrdao-1.2.2/xdao/TextEdit.cc	2008-03-07 11:14:07.000000000 +0000
@@ -21,6 +21,7 @@
 
 #include <stddef.h>
 #include <ctype.h>
+#include <cstring>
 
 TextEdit::TextEdit(const char *sample) : Gtk::Entry()
 {

